addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -335,7 +335,7 @@ def _timesheet_preprocess_get_accounts(self, vals):
         if not project:
             return {}
         company = self.env['res.company'].browse(vals.get('company_id'))
-        mandatory_plans = self._get_mandatory_plans(company, business_domain='timesheet')
+        mandatory_plans = [plan for plan in self._get_mandatory_plans(company, business_domain='timesheet') if plan['column_name'] != 'account_id']
```

Comment:
can't we use [filtered](https://www.odoo.com/documentation/master/developer/reference/backend/orm.html#odoo.models.Model.filtered) here?
I think it would look better, it would looks less like re-inventing the wheel. What do you think? 
(applies to the diff below as well btw)

Reply:
I don't think we can use `filtered` unfortunately because `_get_mandatory_plans()` returns a list of dicts :grimacing: